index.html

```diff
@@ -4,44 +4,158 @@
+          <ul  class = "card-button">
+            <li class="btn btn-primary btn-lg">HTML</li>
+               <li class="btn btn-primary btn-lg">Bootstrap</li>
+           <li class="btn btn-primary btn-lg">Ruby</li></ul>
+        <button class = "projects btn-primary btn-lg">See project</button>  
+        </div>
+        <div class="cardworks1" id="cards">
+            <div class = "cardlist" id="cwaction1">
+              <div class="card-image2" id="card-image"></div>
+               <h3 class="card1">Human Resource Analysis</h3></div>
+               <p class="cardtext">Profesional Art Printing DataA daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>   <ul  class = "card-button">     <li class="btn btn-primary btn-lg">HTML</li>        <li class="btn btn-primary btn-lg">Bootstrap</li>        <li class="btn btn-primary btn-lg">Ruby</li></ul>     <button class = "projects btn-primary btn-lg">See project</button>       </div>     <div class="cardworks2" id="cards">         <div class = "cardlist" id="cwaction1">           <div class="card-image3" id="card-image"></div>             <h3 class="card1">Digital Marketing</h3></div>             <p class="cardtext">Profesional Art Printing DataA daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>           <ul  class = "card-button">             <li class="btn btn-primary btn-lg">HTML</li>                <li class="btn btn-primary btn-lg">Bootstrap</li>                <li class="btn btn-primary btn-lg">Ruby</li></ul>             <button class = "projects btn-primary btn-lg">See project</button>               </div>             <div class="cardworks3" id="cards">       <div class = "cardlist" id="cwaction1">         <div class="card-image4" id="card-image"></div>           <h3 class="card1">Graphic Design</h3></div>           <p class="cardtext">Profesional Art Printing DataA daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>         <ul  class = "card-button">           <li class="btn btn-primary btn-lg">HTML</li>              <li class="btn btn-primary btn-lg">Bootstrap</li>              <li class="btn btn-primary btn-lg">Ruby</li></ul>           <button class = "projects btn-primary btn-lg">See project</button>             </div>           <div class="cardworks4" id="cards">               <div class = "cardlist" id="cwaction1">                 <div class="card-image5" id="card-image"></div>                   <h3 class="card1">Building Design & Construction</h3></div>                   <p class="cardtext">Profesional Art Printing DataA daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>   <ul  class = "card-button">
```

Comment:
- [OPTIONAL] As you need to follow best practices to find your first job in the future, kindly give format to your code to be more readable for the visitors 👍🏼